src/Core/src/Handlers/Image/ImageHandler.cs

```diff
@@ -26,6 +26,10 @@ public partial class ImageHandler : IImageHandler
 			[nameof(IImage.Aspect)] = MapAspect,
 			[nameof(IImage.IsAnimationPlaying)] = MapIsAnimationPlaying,
 			[nameof(IImage.Source)] = MapSource,
+#if GTK
+			["WidthRequest"] = MapWidthRequest,
```

Comment:
Couldn't you use nameof like other properties here?

Reply:
nope we didn't have IImage.WidthRequest and IImage.HeightRequest so that's why we chose to implement them. But let me check maybe there is something better I can do with this.

Reply:
Check some more generic interfaces like IView or IVisualElement or something

Reply:
@webwarrior-ws No, they don't have this either. If you recall Shape was just calling Width Handler and Height Handler every time it got WidthRequest and we couldn't do this so this is why we chose to explicitly use WidthRequest and HeightRequest. I'm thinking I can override the Width Handler and Height handler for images somehow. This way I won't need to define WidthRequest and HeightRequest.

Reply:
@knocte Yes, but it doesn't matter. IView and IImage should have something to be able to use it in the Handlers (VisualElement.cs is in Controls). I'm working on another solution right now.

Reply:
what are you talking about? Afshin requested here the use of nameof, which is not a request to change behaviour; this is a cosmetic issue

Reply:
Well we can't use nameof. And this got me thinking that the implementations is not right. I committed a WIP commit which is still right and I think is more consistent with previous implementation. @knocte @aarani @webwarrior-ws can you guys take a look? if you don't like it I can just drop that commit.

Reply:
I mean we can't use nameof for WidthRequest and HeightRequest.

Reply:
by "previous implementation" I mean what was implemented before this PR.

Reply:
also the use of WIP for that commit is not right. it's tested and done. just let me know which one do you think better:
1. squashing the commit and getting rid of "WidthRequest" and doing what other widgets are doing. (This is my preferred option)
2. dropping the commit and keeping the "WidthRequest"

Reply:
I want to summon @aarani here because I think @parhamsaremi is not understanding the point of this convo